client/TracyProfiler.cpp

```diff
@@ -234,7 +270,8 @@ static int64_t SetupHwTimer()
     CpuId( regs, 0x80000007 );
     if( !( regs[3] & ( 1 << 8 ) ) )
     {
-        const char* noCheck = getenv( "TRACY_NO_INVARIANT_CHECK" );
+        char buffer[32];
+        const char* noCheck = GetEnvVar( "TRACY_NO_INVARIANT_CHECK", buffer, sizeof(buffer) / sizeof(buffer[0]) );
```

Comment:
I'd prefer to keep the `getenv()` interface, so that this silly buffer business can be omitted. You can just use static char buffer in `GetEnvVar()`, if needed.